std/algebra/twistededwards/bandersnatch/point_test.go

```diff
@@ -351,25 +361,39 @@ func TestNeg(t *testing.T) {
 
 }
 
-// benches
+// Bench
+func BenchmarkDouble(b *testing.B) {
```

Comment:
mmhh I see what we do here, not very go-idiomatic (ie the benchmark is going to be run many times).

One way is to add this to the go test command line: ` -benchtime=1x`

Reply:
Got rid of benchmarks. Can be ran with `api.Counter` when needed.